csp_billing_adapter/csp_config.py

```diff
@@ -27,7 +27,16 @@
 def create_csp_config(
     hook,
     config: Config,
-):
+) -> None:
+    """
+    Initialise the csp_config data store.
```

Comment:
Nitpick: I think we would go with the spellings the doc/marketing team has agreed on. Not sure off the top of my head if that's with z's or s's.

Reply:
British english is my first language so I frequent use the `ise` rather than the `ize` spellings ;-)

Reply:
Not a big issue imo. Especially since it's not part of docs :shrug: